### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by shortland** [mailyns/NooksMarketplace#1]
Will this be a separate third party website/marketplace for users to add each other & list items for sale?

I would definitely love to collaborate if you'd like!

**Comment by mailynhuynh** [comment]
Yes, this will be a third party website. I'd love to collaborate if you are still interested in Nook's Marketplace!

**Comment by shortland** [comment]
Cool I submitted a pull request and wrote-up a bit there. Read it when you can @mailyns
